pkg/appStore/installedApp/read/bean/bean.go

```diff
@@ -0,0 +1,71 @@
+package bean
+
+type InstalledAppMin struct {
+	// Installed App details
+	Id     int
+	Active bool
+	// Deprecated; currently in use for backward compatibility
+	GitOpsRepoName string
+	// Deprecated; use deployment_config table instead GitOpsRepoName has been migrated to GitOpsRepoUrl; Make sure to migrate from GitOpsRepoName for future flows
+	GitOpsRepoUrl      string
+	IsCustomRepository bool
+	// Deprecated; use deployment_config table instead
+	DeploymentAppType          string
+	DeploymentAppDeleteRequest bool
+	EnvironmentId              int
+	AppId                      int
+}
+
+type InstalledAppWithAppDetails struct {
+	InstalledAppMin
```

Comment:
try to keep all nested objects as pointers

Reply:
updated 👍